sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -496,7 +496,9 @@ fromStatementBody
 querySpecification
     : transformClause
       fromClause?
-      whereClause?                                                          #transformQuerySpecification
+      whereClause?
+      aggregationClause?
+      havingClause?                                                       #transformQuerySpecification
```

Comment:
Could you update the SQL doc, too?

Reply:
> Could you update the SQL doc, too?

Yea.

Reply:
> Could you update the SQL doc, too?

Can we add this after all things done? and we need to add a new page like `Where clause`